MaxText/configs/base.yml

```diff
@@ -241,3 +241,7 @@ decode_sampling_temperature: 1.
 
 eval_interval: -1  # the specific number of train step between eval_step
 target_eval_loss: 0.  # early stop once reaching target eval_loss
+
+# Goodput parameters
+enable_goodput_record: False
```

Comment:
I would prefer "enable_goodput_recording" and "enable_goodput_calculation" but I will leave that to you and Rafi to sort out.

Reply:
@rwitten, let me know your thoughts.

Reply:
Updated per your recommendation @xuefgu!